kickstart/PXELinux.erb

```diff
@@ -37,7 +37,11 @@ DEFAULT linux
 LABEL linux
     KERNEL <%= @kernel %>
     <% if @host.operatingsystem.name == 'Fedora' and @host.operatingsystem.major.to_i > 16 -%>
-    APPEND initrd=<%= @initrd %> ks=<%= foreman_url('provision')%> ks.device=bootif network ks.sendmac <%= options %>
+      <% if @host.operatingsystem.name.match(/.*atomic.*/i) %>
```

Comment:
Is this meant to be within the Fedora block?  The other templates appear to apply to other OSes.